packages/language/src/index.js

```diff
@@ -434,3 +434,4 @@ Language.prototype.text = function(content, options) {
 };
 
 module.exports = Language;
+module.exports.v1beta1 = require('./v1beta1');
```

Comment:
As the initial step, I think we can skip comments here -- so it exists but nobody would notice. That's not our final goal but won't cause any problems.
Then let me look into the doc scripts for #1492, which may end up with more edits here, but I think that would be acceptable.